### ISSUE TITLE
Add zIndex to webmaps

### ISSUE THREAD

**Issue by DEQrmichie** [donco/odeqstatusandtrends#35]
Depending on how the different layers are clicked on or off they  can become overlapped. This can be addresed by adding a map pane with a zindex. Layers on the top have the highest zIndex. I suggest we order the layers in the following way:

```r 
map <- leaflet::leaflet() %>%
    leaflet::addTiles() %>%
    leaflet::addMapPane("Tiles", zIndex = 420) %>%
    leaflet::addMapPane("hydroTiles", zIndex = 425) %>%
    leaflet::addMapPane("assessment_area", zIndex = 430) %>%
    leaflet::addMapPane("agwqma", zIndex = 440) %>%
    leaflet::addMapPane("IRpolygons", zIndex = 450) %>%
    leaflet::addMapPane("IRpolylines", zIndex = 460) %>%
    leaflet::addMapPane("Status_polygons", zIndex = 470) %>%
    leaflet::addMapPane("Status_polylines", zIndex = 480) %>%
    leaflet::addMapPane("Status_points", zIndex = 490)
```

To add a specfic layer into a pane the code looks something like this:

```r
    map <- map %>%
      leaflet::addPolygons(data = wql_streams_ws_shp,
                           opacity = 1,
                           weight = 3.5,
                           color = "red",
                           fillColor = "red",
                           fillOpacity = 0.25,
                           popup = ~paste0("<b>", AU_Name,
                                           # "<br>Parameter:</b> ", Char_Name,
                                           "<br></b><br>",
                                           sapply(AU_ID, WQLpopupTable, USE.NAMES = FALSE)),
                           popupOptions = leaflet::popupOptions(maxWidth = 600, maxHeight = 300),
                           highlightOptions = leaflet::highlightOptions(color = "black", weight = 8, opacity = 1),
                           options = leaflet::leafletOptions(pane = "IRpolygons"),
                           label = ~AU_Name,
                           smoothFactor = 1.5,
                           group = "WQ Listed Assessment Units")
```

**Comment by DEQcdonald** [comment]
Addressed in commit: 448c4b3890bae64ecf54b2c6ee74ab15f4493bd6
Confirm on re-run

**Comment by DEQcdonald** [comment]
Confirmed.
